crypto/bls_multisig.go

```diff
@@ -72,6 +73,9 @@ func BLSVerifyPOP(pk PublicKey, s Signature) (bool, error) {
 // The order of the signatures in the slice does not matter since the aggregation
 // is commutative. The slice should not be empty.
 // No subgroup membership check is performed on the input signatures.
+// Expected error returns during normal operations:
+//  - invalidInputsError if no signatures are provided (sigs is empty) or
+//    some signatures are invalid
```

Comment:
Just clarifying what invalid means.
```suggestion
// Expected error returns during normal operations:
//  - invalidInputsError if no signatures are provided (sigs is empty) or
//    at least one signature fails to deserialize.
```

Reply:
suggestion included in https://github.com/onflow/flow-go/pull/1830/commits/42a51d24cee91f35b355670231e72f7f9aec52c5